loadgen/scenario.go

```diff
@@ -78,10 +88,15 @@ type RunOptions struct {
 	Configuration RunConfiguration
 	// ScenarioOptions are options passed from the command line. Do not mutate these.
 	ScenarioOptions map[string]string
+	// The namespace that was used when connecting the client.
+	Namespace string
+	// Unique salt for this scenario run. Use to provide values that are unique but stable for
+	// the duration of the scenario run.
+	Salt uuid.UUID
```

Comment:
```suggestion
	ScenarioID string
```

I think this would be clearer to users, or maybe `LocalRunID` or maybe `ExecutionID` or something. Salt is a confusing term for this use IMO.

But really, is this needed? Can't users just create a UUID if they need it?

Reply:
The reason is to avoid counting things when multiple runs have been made on the same cluster. This is very easy to have happen right now since run-id is provided on the CLI and may very easily take the same value repeatedly.